src/data/navigation/header.js

```diff
@@ -16,6 +16,10 @@ module.exports = [
         }
       ]
     },
+    {
+      title: "Feed Ingestion",
```

Comment:
```suggestion
      title: "REST",
```

Move this to be after GraphQL